src/Tags/ResponsiveTag.php

```diff
@@ -92,4 +92,26 @@ private function getAttributeString(): string
                 return $name . '="' . $value . '"';
             })->implode(' ');
     }
+
+    private function includePlaceholder(): bool
+    {
+        $includePlaceholder = $this->params->get('placeholder');
+
+        if (isset($includePlaceholder)) {
```

Comment:
This will always return true, since you just set the variable above, better to use `is_null`

Reply:
I tested both `isset` and `is_null` and there's a problem with `is_null`.

Let's say you have `placeholder => true` in the config file and want to override this on the tag with `placeholder="false"`;  `is_null` will ignore this, as `is_null(false)` returns `false`.

I use `isset` to always prefer what you set on the tag over what's in the config.

Reply:
You could do `if ($this->params->has('placeholder'))` instead as a check